src/corrgi/dask.py

```diff
@@ -23,7 +23,10 @@ def perform_auto_counts(catalog: Catalog, *args) -> np.ndarray:
         The histogram with the sample distance counts.
     """
     # Get counts between points of different partitions
-    alignment = autocorrelation_alignment(catalog.hc_structure)
+    bins = args[0].make_bins()
+    if len(bins) == 2:
+        bins = bins[0]
```

Comment:
When using the projected correlation, these will be the bins in projected separation [Mpc/h]. Does the algorithm above still hold?